### DISCUSSION TITLE
Re-Activate License Key

### DISCUSSION THREAD

**Discussion by magickiat** [usebruno/bruno#1979] [Q&A] [answered]
I has activated a key on another computer and it broken.
When I've re-install OS and I need to use activated key.
For now, there is a way to do that?

Thanks you.

**Comment by helloanoop** [comment, marked as answer]
@magickiat 

Can you email the license that you need to deactivate to support@usebruno.com
We will deactivate it, and then you can reactivate it on your new device.

In the future, we will provide a self-serve portal which can be used to login via email and deactivate the keys by yourself.

Best,
Anoop
